api/serviceregistry/helper.go

```diff
@@ -16,20 +16,6 @@ func (e validationErrors) Error() string {
 	return b.String()
 }
 
-func (x *ServiceRequest) Validate() error {
```

Comment:
not sure why you deleted this. it is no longer needed?

Reply:
We no longer have ServiceRequest object

Reply:
yes, that's right, it was used in QueryService and that was deleted. I get it now, thanks for the explanation